### ISSUE TITLE
Use bambi for MMMs

### ISSUE THREAD

**Issue by ricardoV94** [pymc-labs/pymc-marketing#398]

**Comment by ricardoV94** [comment]
> Bambi + MMM? Interesting... Bambi + CLV would be ... heaven.

For CLV, I assume this is only relevant for models that allow predictors?

**Comment by zwelitunyiswa** [comment]
Exactly. Would love to be able to add predictors or allow for a correlation between the purchase and attrition process.

**Comment by ricardoV94** [comment]
Okay that's a very good thing to keep in mind when we implement such CLV models!

**Comment by wd60622** [comment]
Brainstorming how this might look
```
y ~ 1 + menten(geometric(Google + Facebook, max_lag=8))
